### DISCUSSION TITLE
essay/2024/09/30/geek-news

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#235] [Comments]
# essay/2024/09/30/geek-news

긱뉴스를 처음 보기 시작했던 게 2019년이었던가. 서비스가 나오자마자 보기 시작했습니다. 서비스 자체보다는 서비스를 어떻게 운영하는지에 더 관심이 있었습니다.

https://jeho.page/essay/2024/09/30/geek-news.html

**Comment by yeonhoyoon** [comment]
꾸준함 대단하네요.

**Reply by BenjaminKim** [reply]
@yeonhoyoon 우리도 꾸준히 재밌게 해보자!

**Comment by porporpor020202** [comment]
큰 도움이 되는 글입니다.
공유해주셔서 감사합니다.
